scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -469,27 +493,35 @@ <h6>Phenotype terms (HPO):</h6>
     </div>
     <div class="form-group" id="chromosome_search">
       <div class="row" style="margin-top:20px;">
-        <div class="col-1" style="margin-right:3em">
+        <div class="col-1 offset-2">
           {{ wtf.form_field(form.chrom) }}
         </div>
-
-        <div class="col-md-2 col-lg-2 col-xl-1">
+        <div class="col-2">
           {{ wtf.form_field(form.start) }}
         </div>
-        <div class="col-md-2 col-lg-2 col-xl-1" style="margin-right:3em">
+        <div class="col-2">
           {{ wtf.form_field(form.end) }}
         </div>
-
-        <div class="col-md-3 col-lg-3 col-xl-2">
+        <div class="col-2">
           {{ wtf.form_field(form.cytoband_start) }}
         </div>
-        <div class="col-md-3 col-lg-3 col-xl-2">
+        <div class="col-2">
           {{ wtf.form_field(form.cytoband_end) }}
         </div>
-        <div class="col-md-1 col-lg-1 col-xl-5" id="empty_space"></div>
+      </div>
+      <div class="row">
+        <div class="col-2">
```

Comment:
Any chance of having this on the row before for cancer, sort of next to the chromosome field again? The overall layout isn't quite as elegant as it was in the previous version anymore. 😊

Reply:
Yes, I agree, I'll revert to how it was before!